rails_docker/.hooks/commit-msg.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Not sure about the file naming. `commit-message-format.sh` would be more informative

Reply:
@olivierobert I follow the naming from the hook name https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks (commit-msg). Because we can add more action in this hook too.